### PR TITLE
Typos and some slight copy changes

### PR DESCRIPTION
**Pull request by roblofthouse**
More typos fixed and some slight copy changes.

### CODE REVIEW

File: content/docs/01-what-why-and-how.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ When `setRemoteDescription` has been called on both sides, the WebRTC Agents now
 
 #### `ontrack`
 
-`ontrack` is a callback that is fired when a RTP packet is received from the remote peer. The incoming packets would have been declared in the Session Description that was passed to `setRemoteDescription`
+`ontrack` is a callback that is fired when an RTP packet is received from the remote peer. The incoming packets would have been declared in the Session Description that was passed to `setRemoteDescription`
```
Comment (mogren) [inline content/docs/01-what-why-and-how.md:141]: Nit. We should add this sentence with a punctuation... `.`

File: content/docs/07-data-communication.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ If the data channel type is `DATA_CHANNEL_PARTIAL_RELIABLE`, the suffixes config
 * `TIMED` - Defines for how long time (in ms) the sender will re-send the message before giving up.
 
 #### Label
-The name of the data channel as a UTF-8-encoded string. This may be an empty string.
+The name of the data channel is a UTF-8-encoded string. This may be an empty string.
```
Comment (mogren) [inline content/docs/07-data-communication.md:81]: This sentence still doesn't really feel right. How about something like this:
```
A UTF-8-encoded string containing the name of the data channel. This string may be empty.
```
The emphasis being that it's UTF-8 and not ASCII.

**Review by mogren — APPROVED** [review]
Thanks again @roblofthouse! 

I just had two minor comments, address them if you feel like it.

File: content/docs/07-data-communication.md (hunk 2 above)
Comment (roblofthouse) [reply content/docs/07-data-communication.md:81]: @mogren Ha, ah yes. I'll fix.

File: content/docs/01-what-why-and-how.md (hunk 1 above)
Comment (roblofthouse) [reply content/docs/01-what-why-and-how.md:141]: I'll add that as part of this PR. There are a few other punctuation errors from the original copy but I'll fix them in another PR.

File: content/docs/05-real-time-networking.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ A longer jitter buffer delay means your network is highly congested.
 
 ## Detecting Congestion
 Before we can even resolve congestion, we need to detect it. To detect it we use a congestion controller. This is a complicated subject, and is still rapidly changing.
-New algorithms are still be published and tested. At a high level they all operate the same. A congestion controller provides a bandwidth estimates given some inputs.
+New algorithms are still be published and tested. At a high level they all operate the same. A congestion controller provides bandwidth estimates given some inputs.
```
Comment (roblofthouse) [inline content/docs/05-real-time-networking.md:155]: Just noticed this one as well actually. It should be "New algorithms are still being published and tested"

I'll push a fix.

File: content/docs/01-what-why-and-how.md (hunk 1 above)
Comment (roblofthouse) [reply content/docs/01-what-why-and-how.md]: Punctuation added.

**Comment by roblofthouse** [comment]
@mogren I've addressed your comments and also made two more minor typo fixes that I missed. Feel free to merge if you are happy :)

File: content/docs/07-data-communication.md (hunk 2 above)
Comment (roblofthouse) [reply content/docs/07-data-communication.md]: I've changed it to the suggested sentence.

**Comment by mogren** [comment]
@roblofthouse Awesome, thank you!

**Pull request merged**